teuthology/orchestra/test/test_raise_for_status.py

```diff
@@ -0,0 +1,62 @@
+from io import BytesIO
+
+from mock import MagicMock, Mock
+from pytest import raises
+
+from teuthology.orchestra.run import RemoteProcess
+from teuthology.exceptions import (CommandCrashedError, CommandFailedError,
+                                   ConnectionLostError)
+
+
+class TestRaiseForStatus(object):
+    def setup(self):
+        m_transport = MagicMock()
+        m_transport.getpeername.return_value = ('name', 22)
+        self.m_ssh = MagicMock()
+        self.m_ssh.get_transport.return_value = m_transport
+        self.stdout = BytesIO(b'test_hostname')
```

Comment:
Good use of the mock setup, aligned with https://github.com/ceph/teuthology/blob/master/teuthology/orchestra/test/test_run.py